libfwupd/fwupd-bios-attr.c

```diff
@@ -546,6 +645,10 @@ fwupd_bios_attr_from_key_value(FwupdBiosAttr *self, const gchar *key, GVariant *
 		fwupd_bios_attr_set_path(self, g_variant_get_string(value, NULL));
 		return;
 	}
+	if (g_strcmp0(key, FWUPD_RESULT_KEY_BIOS_ATTR_POSITIVE_VALUE) == 0) {
+		fwupd_bios_attr_set_positive_value(self, g_variant_get_string(value, NULL));
```

Comment:
not sure i like the `positive` noun. Got any other ideas?

Reply:
OK so if the heuristic to set it is dropped from `fu-bios-attrs`, how about `preferred`?  Then we can set preferred only with security attributes.

Reply:
`preferred` makes sense to me. That or `ideal` or `endorsed` or `recommended`